jupyterhub_traefik_proxy/proxy.py

```diff
@@ -143,7 +143,7 @@ def _launch_traefik(self, config_type):
                         "traefik",
                         "--etcd",
                         "--etcd.useapiv3=true",
-                        "--etcd.username=root",
+                        "--etcd.username=" + self.etcd_username,
                         "--etcd.password=" + self.etcd_password,
```

Comment:
Ah, I didn't catch this before. I think it's actually important to not set credentials on the command-line (other processes can see these values). Let's switch this to writing a private toml config file.

Reply:
You're right @minrk. I've just added some changes addressing this.
```TraefikEtcdProxy``` will now load its static configuration (including the etcd credentials) from a toml file just like ```TraefikTomlProxy```.